project/npda/models/npda_user.py

```diff
@@ -85,7 +85,7 @@ def create_superuser(self, first_name, last_name, email, password):
             is_rcpch_audit_team_member=True,
             is_rcpch_staff=True,
             email_confirmed=True,
-            view_preference=2 # national
+            view_preference=1
```

Comment:
quality of life improvement for the default superuser, without this they wouldn't see the CSV upload UI by default